### ISSUE TITLE
[Table] Scroll when reordering columns

### ISSUE THREAD

**Issue by speletz** [palantir/blueprint#1585]
### Bug report

- __Package version(s)__: blueprintjs/table: 1.18.04.3.0
- __Browser and OS versions__: Chrome 60.0.3112.113 and MacOS 10.12.6

#### Steps to reproduce

1. Create a table with many columns so that you are not able to see all columns without scrolling left/right.
2. Drag a column from it's existing position to a position that requires scrolling left/right. For example, attempt to move the rightmost column to the leftmost position. 

#### Actual behavior

I have to drag and drop the column, scroll left/right, drag and drop the column again, etc. until I get the column where I want it. 

#### Expected behavior

I would expect the table to scroll when reordering columns.

**Comment by cmslewis** [comment]
@speletz Yep, thanks for filing. 👍  We got this working with drag-selection recently, but scrolling while reordering is still on deck.
